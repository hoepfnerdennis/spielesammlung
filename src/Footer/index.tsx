import React from 'react';
import styles from './styles.module.css';

const Footer = (): JSX.Element => {
  return <footer className={styles.footer}>made with ❤︎ by Selina & Dennis</footer>;
};

export default Footer;
