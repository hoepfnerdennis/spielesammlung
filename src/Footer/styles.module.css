.footer {
    box-sizing: border-box;
    height: 100px;
    padding: 1rem;

    background-color: darkcyan;
    color: white;

    display: flex;
    align-items: flex-end;
    justify-content: center;
}