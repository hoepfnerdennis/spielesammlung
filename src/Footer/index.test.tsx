import React from 'react';
import { render } from '@testing-library/react';
import Footer from './index';

describe('Footer', () => {
  it('should render and match snapshot', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
