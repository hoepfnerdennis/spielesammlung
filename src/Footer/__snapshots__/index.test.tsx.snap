// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render and match snapshot 1`] = `
<div>
  <footer
    class="footer"
  >
    made with ❤︎ by Selina & Dennis
  </footer>
</div>
`;
